lib/fpm/package/deb.rb

```diff
@@ -83,6 +83,9 @@ class FPM::Package::Deb < FPM::Package
 
   option "--group", "GROUP", "The group owner of files in this package", :default => 'root'
 
+  option "--no-changelog", :flag,
```

Comment:
I prefer positive terms (remove the word 'no')

I think this should just be `--changelog` (a positive term) with `:default => true` and users can disable with `--no-deb-changelog`.

Reply:
@jordansissel is this comment still valid since as `changelog` option is present already.